src/perf-goggles.js

```diff
@@ -1,134 +1,179 @@
 import 'isomorphic-fetch';
-import isEqual from 'lodash.isequal';
+import isEqual from 'lodash/isEqual';
```

Comment:
Using `lodash.isequal` is a smaller package:
https://www.npmjs.com/package/lodash.isequal

Please revert.